manager/src/main/scala/org/openinfralabs/caerus/cache/manager/BasicStorageIOMultiTierPlanner.scala

```diff
@@ -2,16 +2,17 @@ package org.openinfralabs.caerus.cache.manager
 
 import org.apache.hadoop.fs.Path
 import org.apache.spark.sql.catalyst.plans.logical.{Filter, Project}
+import org.openinfralabs.caerus.cache.common.Tier.Tier
 import org.openinfralabs.caerus.cache.common.plans._
-import org.openinfralabs.caerus.cache.common.{Caching, Candidate, FileSkippingIndexing, Repartitioning}
+import org.openinfralabs.caerus.cache.common.{Caching, Candidate, FileSkippingIndexing, Repartitioning, Tier}
```

Comment:
You have imported both Tier.Tier and Tier. You need only one.

Reply:
I see, so I need to use Tier.Tier